tests/geometric-multiscale/AxialGeoMultiscale.cpp

```diff
@@ -0,0 +1,96 @@
+#ifndef PRECICE_NO_MPI
+
+#include "testing/Testing.hpp"
+
+#include <precice/Participant.hpp>
+#include <vector>
+#include "testing/TestContext.hpp"
+
+using namespace precice;
+using precice::testing::TestContext;
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(GeometricMultiscale)
+BOOST_AUTO_TEST_CASE(AxialGeoMultiscale)
+{
+  PRECICE_TEST("SolverOne"_on(1_rank), "SolverTwo"_on(1_rank))
```

Comment:
```suggestion
  PRECICE_TEST("Fluid1D"_on(1_rank), "Fluid3D"_on(1_rank))
```

or similar could make things easier to read?

Reply:
We generally only use very generic names in tests, but indeed, this would make more sense. I would then also rename the data to something velocity-related.